manifests/early-starts.yaml

```diff
@@ -0,0 +1,14 @@
+name: Early Starts
+authors: jmelesky
+homepage: https://codeberg.org/jmelesky/endless-sky-early-starts
+license: CC-BY-SA-4.0
+version: 0.9
+shortDescription: Start the game at one of five earlier dates
+description: Real simple. Start the same way, in the same place, but earlier.
+
+Why? The big reason would be more time before the plot kicks off (which is date-based). You can use that time to get started on quests that are unavailable after the plot starts, or to prep yourself well for the main plot, or just to play for years without having to worry about the plot.
+
+url: https://codeberg.org/jmelesky/endless-sky-early-starts/archive/release-0.9.zip
+autoupdate:
+  type: tag
+  url: https://codeberg.org/jmelesky/endless-sky-early-starts/archive/release-$version.zip
```

Comment:
the tag is named `release-0.9`, which is what the autoupdate finds as `$version`. Thus, the new url would become `https://codeberg.org/jmelesky/endless-sky-early-starts/archive/release-release-0.9.zip`. Either change the tag name to not include `release-` or the manifest's version to include it.